<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EvenOdd</class>
 <widget class="QMainWindow" name="EvenOdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:reflect, x1:1, y1:1, x2:0, y2:0, stop:0.179104 rgba(148, 96, 255, 255), stop:0.79602 rgba(99, 252, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_throw">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 85, 255);</string>
    </property>
    <property name="text">
     <string>Бросить кубики</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei Light</family>
      <pointsize>11</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(150, 137, 241);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_stavka">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(150, 137, 241);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(131, 137, 241);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>Введите вашу ставку:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_strt">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 85, 255);</string>
    </property>
    <property name="text">
     <string>Сделать ставку и начать игру</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 85, 255);</string>
    </property>
    <property name="text">
     <string>Вернуться в главное меню</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_balance">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(150, 137, 241);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalWidget_3" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>361</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioButton_even">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <underline>false</underline>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(131, 137, 241);</string>
       </property>
       <property name="text">
        <string>ЧЕТНОЕ</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_odd">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <underline>false</underline>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(131, 137, 241);</string>
       </property>
       <property name="text">
        <string>НЕЧЕТНОЕ</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>462</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
